index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Carter+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Zellij</title>
+  </head>
+  <body>
+    <header class="interface">
+      <h1 class="game-title">Zellij</h1>
+
+      <form id="create-new-game" class="player-info">
+        <div class="player-name">
+          <label class="sr-only" for="nameone">Player 1:</label>
+          <input
+            class="player-name-input"
+            type="text"
+            id="nameone"
+            name="player_name"
+            placeholder="Player 1"
+          />
+        </div>
+        <div class="player-name">
+          <label class="sr-only" for="nametwo">Player 2:</label>
+          <input
+            class="player-name-input"
+            type="text"
+            id="nametwo"
+            name="player_name"
+            placeholder="Player 2"
+          />
+        </div>
+        <button class="start-two-players" type="submit">
+          <span>New Game</span>
+        </button>
+      </form>
+    </header>
+    <section class="game-wrapper">
+      <canvas class="game-surface" width="1280" height="720"></canvas>
+      <button class="help">
+        <img src="images/help.svg" width="35px" height="35px" alt="Help" />
+      </button>
+    </section>
+    <footer class="footer">
+      <small class="attribution">
+        <p>
+          Original Gameplay:
+          <a
+            href="https://planbgames.com/games/azul-next-move-games-michael-kiesling-strategy-abstract-board-game-winner-spiel-des-jahres-game-of-the-year-cannes-portuguese-tiles-royal-palace-of-evora-1"
+            >Azul</a
+          >
+          by Michael Kiesling / Next Move Games
+        </p>
+        <p>
+          <a href="https://maaot.itch.io/mossy-cavern">Blue Magician</a> by
+          maaot at itch.io
+        </p>
+        <p>
+          &copy;2020 Programming & Design by Victor Nastasa — 1st Ironhack
+          Project
+        </p>
+      </small>
+    </footer>
+
+    <script type="text/javascript" src="js/help.js"></script>
+    <script type="text/javascript" src="js/tiles.js"></script>
+    <script type="text/javascript" src="js/index.js"></script>
+    <script type="text/javascript" src="js/interface.js"></script>
+    <script type="text/javascript" src="js/player.js"></script>
```

Comment:
Next step is using [ES Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and bundler to concatenate  and minimise scripts. Check [Rollup]( https://www.rollupjs.org/guide/en/), as the most awesome bundler. ;)